tests/integ/test_pytorch_train.py

```diff
@@ -38,6 +38,8 @@ def fixture_training_job(sagemaker_session, pytorch_full_version, cpu_instance_t
 
 @pytest.mark.canary_quick
 @pytest.mark.regional_testing
+@pytest.mark.gamma_dlc
+@pytest.mark.skipif(PYTHON_VERSION == "py2", reason="PyTorch Inference not supporting Python2.")
```

Comment:
nits:
* s/not supporting/doesn't support
* s/Python2/Python 2

also, it may help to add the version in the reason string, as previous versions of PyTorch did support Python 2